package.json

```diff
@@ -62,7 +62,7 @@
   "license": "MIT",
   "dependencies": {
     "aw-liner": "0.0.2",
-    "aw-parser": "0.0.3",
+    "aw-parser": "0.0.8",
```

Comment:
Should we add a test to catch this sort of issues in future release?

Reply:
I've added a test to `aw-parser`: https://github.com/afterwriting/aw-parser/pull/13/files#diff-7bf59ee18e32a02823031db5b25b384c

Maybe it's worth adding an acceptance test for regression?